exporter/kinesisexporter/tenant.go

```diff
@@ -0,0 +1,56 @@
+package kinesisexporter
+
+import (
+	"fmt"
+
+	"go.opentelemetry.io/collector/consumer/pdata"
+)
+
+const (
+	serviceKey = "service_id"
```

Comment:
Hmm, pretty sure we won't be able to merge these changes back upstream. Is this mostly for debugging our current issues?

Reply:
Yea we won't be merging this upstream but the progress of that has been stagnant so I thought I might as well add it in here. Can yeet it later, alternatively, we could have a branch that we use ourselves that's not master, and I can change obsvs-ingestion to use that branch instead (if that's possible).